packages/lesswrong/server/sqlConnection.ts

```diff
@@ -177,8 +177,10 @@ const logIfSlow = async <T>(
     // eslint-disable-next-line no-console
     console.log(`Finished query #${queryID} (${milliseconds} ms) (${JSON.stringify(result).length}b)`);
   } else if (SLOW_QUERY_REPORT_CUTOFF_MS >= 0 && milliseconds > SLOW_QUERY_REPORT_CUTOFF_MS && !quiet && !isAnyTest) {
-    // eslint-disable-next-line no-console
-    console.trace(`Slow Postgres query detected (${milliseconds} ms): ${getDescription()}`);
+    const description = isDevelopment ? getDescription(50) : getDescription(200);
```

Comment:
Not sure why this is truncating prod to 200 chars, feels safer to leave it as-is?